tools/elastic/README.md

```diff
@@ -16,6 +16,7 @@ done by getting via bash in the running Docker machine. Example:
 ```bash
 docker-machine ssh default sudo sysctl -w vm.max_map_count=262144
 ```
+> Note: If you are running docker natively on linux, you need only run `sysctl -w vm.max_map_count=262144`
```

Comment:
oooh, actually I think all of this predates Docker for Mac, and if you are not on Linux I'm not even sure how it'll go

Reply:
It worked for me on Fedora 29 : )